docs/source/en/api/utilities.mdx

```diff
@@ -0,0 +1,7 @@
+# Utilities
+
+Utility and helper functions for working with 🤗 Diffusers.
+
+## randn_tensor
+
+[[autodoc]] diffusers.utils.randn_tensor
```

Comment:
Might be worth including the following too:

* numpy_to_pil
* pt_to_pil
* load_image
* export_to_video


I know there are quite a few [here](https://github.com/huggingface/diffusers/blob/main/src/diffusers/utils/__init__.py), I only picked the ones I use the most. 

WDYT?

Reply:
That sounds good! For starters I'll add the ones you listed above, and if there are other utilities we see that get a lot of usage we can add them to the page.